src/build.ts

```diff
@@ -129,19 +130,28 @@ async function emitDom() {
 
   function mergeApiDescriptions(
     idl: Browser.WebIdl,
-    descriptions: Record<string, string>,
+    descriptions: { interfaces: { interface: Record<string, any> } },
   ) {
     const namespaces = arrayToMap(
       idl.namespaces!,
       (i) => i.name,
       (i) => i,
     );
-    for (const [key, value] of Object.entries(descriptions)) {
-      const target = idl.interfaces!.interface[key] || namespaces[key];
-      if (target) {
-        target.comment = value;
+
+    for (const [key, descObject] of Object.entries(
+      descriptions.interfaces.interface,
+    )) {
+      const target = idl.interfaces?.interface?.[key] || namespaces[key];
+
+      if (!target) continue;
+
+      // Set interface/class-level comment
+      if (descObject.__comment) {
+        target.comment = descObject.__comment;
       }
     }
```

Comment:
```suggestion
     for (const [key, target] of Object.entries(namespaces)) {
      const descObject = descriptions.interfaces.interface[key];
      if (!target) continue;

      merge(target, descObject, { optional: true });
    }
```

1. Only namespaces need this loop as MDN doesn't have namespace-specific page type. Interfaces are being handled by the revised `merge()` 👍🏻
2. Would be a bit more efficient to iterate over namespaces than interfaces, because the list of the former is much shorter.